src/Controller/Component/FlashComponent.php

```diff
@@ -18,11 +18,15 @@ class FlashComponent extends Component
      */
     public function __call($name, $args)
     {
-        if ('error' == $name) {
-            $name = 'danger';
+        if ($name === 'danger') {
```

Comment:
Don't see any good reason why anyone would call Flash->danger() instead of Flash->error(). Just because bootstrap chose a silly name for the css class doesn't me one has to use it in php code too. In general no other name besides besides the standard error levels makes sense to me.

Once this is merged i'll replace this component with helper. The element name jugglery done below belongs in View layer imo.